.scalafmt.conf

```diff
@@ -46,3 +46,4 @@ align {
 }
 assumeStandardLibraryStripMargin = true
 onTestFailure = "To fix this, run ./bin/scalafmt from the project root directory"
+rewrite.scala3.convertToNewSyntax = true
```

Comment:
there's a `rewrite` section above, it would need to be moved there.

this is in addition to the question of why do it at all.